### DISCUSSION TITLE
v2.1.0

### DISCUSSION THREAD

**Discussion by VisualEhrmanntraut** [ChefKissInc/SMCRadeonSensors#13] [Announcements]
# What's changed

## Bug fixes

- Incorrect temperature readings were reported on Polaris and older cards.

<hr /><em>This discussion was created from the release <a href='https://github.com/ChefKissInc/SMCRadeonSensors/releases/tag/v2.1.0'>v2.1.0</a>.</em>

**Comment by skippydream** [comment]
This build fixed my 127°C reading peaks bug. Reeeallly appreciate man. That's some great job 😁
